framework/src/main/java/org/apache/drill/test/framework/TestDriver.java

```diff
@@ -402,7 +405,7 @@ public void run() {
   private static void runGenerateScript(DataSource datasource) {
     int exitCode = 0;
     String command = CWD + "/resources/" + datasource.src;
-    LOG.debug("Running command " + command);
+    LOG.info("Running command " + command);
```

Comment:
Since we are running the datagen scripts in parallel, we want to know what else was running when this failure happened. On the other hand console output from different threads could lead to some confusion.

I am inclined to have more information displayed.